binr/r2r/r2r.c

```diff
@@ -217,7 +232,28 @@ int main(int argc, char **argv) {
 		// Manually specified path(s)
 		int i;
 		for (i = opt.ind; i < argc; i++) {
-			char *tf = (argv[i][0] == '/')? strdup (argv[i]): r_str_newf ("%s"R_SYS_DIR"%s", cwd, argv[i]);
+			const char *arg = argv[i];
+			if (*arg == '@') {
+				arg++;
+				eprintf ("Category: %s\n", arg);
+				if (!strcmp (arg, "unit")) {
+					r2r_test_run_unit ();
+					continue;
+				} else if (!strcmp (arg, "fuzz")) {
+					eprintf (".fuzz: TODO\n");
+					continue;
+				} else if (!strcmp (arg, "json")) {
+					arg = "../../test/new/db/json";
```

Comment:
so @thestr4ng3r @radare this assume that r2r is always run from a particular dir, right? :(

Reply:
yes, the database directory and i can probably remove all the `../../test/new/db` in here..

Reply:
i'll rework this in a separate pr for a better proposal

Reply:
nvm fixed in here